Sources/ProjectDescription/DeploymentTarget.swift

```diff
@@ -12,18 +12,22 @@ public enum DeploymentTarget: Codable, Hashable {
     case watchOS(targetVersion: String)
     /// The minimum tvOS version your product will support.
     case tvOS(targetVersion: String)
+    /// The minimum visionOS version and the list of devices your product will support.
+    case visionOS(targetVersion: String, devices: DeploymentDevice)
```

Comment:
Why do you need `DeploymentDevice` here? doesn't visionOS only deploys to the Apple Vision?

Reply:
I did this because apple sets `TARGETED_DEVICE_FAMILY` to `1,2,7` (iphone, ipad, apple vision)... I could go either way if you want to remove it but it seemed like I shouldn't be hard-coding it to that

Reply:
I think the idea is they can get run on iphone too but some apps might not want that? honestly not really sure how the platform works yet... I would prefer to give more control to the user and we can remove it if we determine it's not necessary but I'm not too strongly attached to it.

Reply:
Adding it is simpler than removing as it's not a breaking change

Reply:
Removed